plugins/3MFReader/ThreeMFWorkspaceReader.py

```diff
@@ -41,6 +41,18 @@
 i18n_catalog = i18nCatalog("cura")
 
 
+_ignored_machine_network_metadata = {
+    "um_cloud_cluster_id",
+    "um_network_key",
+    "um_linked_to_account",
+    "host_guid",
+    "removal_warning",
+    "group_name",
+    "group_size",
+    "connection_type"
+}  # type: Set[str]
```

Comment:
This mirrors the `ignore_keys` here (but not quite):
https://github.com/Ultimaker/Cura/pull/8116/files
Wouldn't it be better to have a single list?

Reply:
That shouldn't have been merged (and it has also been reverted). These keys _should_ not have ended up in a project file (but they should be stored normally). If we remove them by means of the version upgrader, they are removed from locally stored printers as well.
So the list is only there for the project loading.